rapid7_insightvm_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,2237 @@
+      title: Last Scan End
+      type: string
+      required: false
+    last_scan_start:
+      description: When the last scan was started
+      title: Last Scan Start
+      type: string
+      required: false
+    mac:
+      description: Media Access Control (MAC) address, e.g. AF:12:BC:5A:F7:48
+      title: MAC
+      type: string
+      required: false
+    malware_kits:
+      description: Number of malware kits
+      title: Malware Kits
+      type: integer
+      required: false
+    moderate_vulnerabilities:
+      description: Number of moderate vulnerabilities
+      title: Moderate Vulnerabilities
+      type: integer
+      required: false
+    os_architecture:
+      description: The srchitecture of the os
+      title:  OS Architecture
+      type: string
+      required: false
+    os_description:
+      description: Description of the os
+      title: OS Description
+      type: string
+      required: false
+    os_family:
+      description: Family of the os
+      title: OS Family
+      type: string
+      required: false
+    os_name:
+      description: Name of the os
+      title: OS Name
+      type: string
+      required: false
+    os_system_name:
+      description: Name of the system os
+      title: OS System Name
+      type: string
+      required: false
+    os_type:
+      description: Type of os
+      title: OS Type
+      type: string
+      required: false
+    os_vendor:
+      description: Vendor of the os
+      title: OS Vendor
+      type: string
+      required: false
+    riskScore:
+      description: Risk score (with criticality adjustments) of the asset
+      title: Risk Score
+      type: float
+      required: false
+    severe_vulnerabilities:
+      description: Number of sever vulns
+      title: Severe Vulnerabilities
+      type: integer
+      required: false
+    tags:
+      description: Asset tags
+      title: Tags
+      type: '[]asset_tag'
+      required: false
+    total_vulnerabilities:
+      description: Total number of vulns
+      title: Total Vulnerabilities
+      type: integer
+      required: false
+    source:
+      description: Source of the asset
+      title: Source
+      type: string
+      required: false
+    unique_identifiers:
+      description: Uniqure aspects of the asset
+      title: Unique Identifiers
+      type: '[]identifiers'
+      required: false
+    new:
+      description: unknown
+      title: New
+      type: '[]string'
+      required: false
+    remediated:
+      description: unknown
+      title: Remediates
+      type: '[]string'
+      required: false
+
+  
+connection:
+  credentials:
+    title: API Key
+    description: API key from account
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  region:
+    title: Region
+    description: the region in which the plugin will work
+    type: string
+    required: true
+    example: us
+  max_pages:
```

Comment:
Do we think we need to have this as part of the connection? Seems like if we want to offer it we should do it at the action level. And I think we should be careful with paging in general (eg asset_search) due to the amount of pages/data that could return